packages/iTwinUI-react/src/core/Header/HeaderDropdownButton.tsx

```diff
@@ -42,29 +43,27 @@ export const HeaderDropdownButton: HeaderDropdownButtonComponent = React.forward
         onShow={() => setIsMenuOpen(true)}
         onHide={() => setIsMenuOpen(false)}
       >
-        <button
+        <HeaderBasicButton
           className={cx('iui-header-breadcrumb-button', className)}
           ref={refs}
           aria-label='Dropdown'
+          endIcon={
+            isMenuOpen ? (
+              <SvgCaretUpSmall
+                className='iui-header-breadcrumb-button-dropdown-icon'
+                aria-hidden
+              />
+            ) : (
+              <SvgCaretDownSmall
+                className='iui-header-breadcrumb-button-dropdown-icon'
+                aria-hidden
+              />
+            )
+          }
           {...rest}
         >
-          {startIcon &&
-            React.cloneElement(startIcon, {
-              className: startIcon.props.className,
-            })}
```

Comment:
The `startIcon` prop isn't destructed in `HeaderDropdownButton` since `HeaderDropdownButton` does not change the `startIcon`, so it is passed to `HeaderBasicButton` through `...rest`